package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "antlr4-tool",
-  "version": "1.0.26",
-  "description": "An Antlr4 tool that generates both Typescript and JavaScript Antlr Parsers",
```

Comment:
Not sure if we want to change the name and version. The reason being is that we might want to add a compatibility option for a version of the antlr4 jar. Please revert these changes, and only increment the version.